### ISSUE TITLE
Fetch land sometimes results in no pop-up to choose land to fetch

### ISSUE THREAD

**Issue by DavidRagone** [magefree/mage#727]
I am not sure what the frequency is of this, but sometimes when I use a fetch-land, my opponent is told the xmage is waiting on me, and I am told that xmage is waiting on my opponent. There appears no way to proceed, and the fetch ability simply sits on the stack. The game log says that I am searching my library, but no library window appears, so I am unable to search. The only way I could proceed with the game was to hit F2, which caused me to effectively fail-to-find.

Please let me know if I can provide more information on this.

**Comment by LevelX2** [comment]
What mana sources are already on the battlefield when you use the fetch land?
Game log would be helpful.

**Comment by dylan1020** [comment]
One thing that you could try is clicking on the ability on the stack. That sometimes brings the window up.

**Comment by fireshoes** [comment]
Not specific to the fetchlands, but relating to the issue of mana sources causing lag, I noticed today it started during a 4-player commander game. It seemed to me that the lag began when I played Contested War Zone, but that could be incorrect.
![edhlands](https://cloud.githubusercontent.com/assets/8553747/6240597/4a8ce00c-b6d6-11e4-8f10-5112c5362100.png)

**Comment by fireshoes** [comment]
Here is another commander game where things got laggy. I think it is the filter lands, Fire-Lit Thicket and Fetid Heath. It got a lot better when the top right player left.

![landlag](https://cloud.githubusercontent.com/assets/8553747/6332693/e99fba32-bb4c-11e4-8385-db33a6e81d37.png)

**Comment by DavidRagone** [comment]
I've had this occur at various points. First fetch land of the game, or further into the game. I have not had a situation where it occurs, and then does not continue to occur.

**Comment by nazimasgarov** [comment]
Yes, something similar happened to me during a game a few days ago. In the same match, my Fated Conflagration's Scry didn't show anything, either.

**Comment by vaidashwin** [comment]
I did some testing on this and found that split cards are at least one possible cause. It looks like if you have a split card in the library, then the library window fails to display and this behavior happens, both for fetches and other tutors (confirmed with Tinker and Demonic Tutor at least with Fire / Ice in the library). I don't see any errors printed to the console, so I'm guessing something is getting eaten. If I remove the split card from the library (even by just sideboarding it out), fetches and tutors work fine.

**Comment by nazimasgarov** [comment]
No, that's not my case. Actually, the opposite is the case: I've used fetches without any problems in my extended deck with Wear/Tear, but the situation I mentioned was in a Standard deck.

**Comment by emerald000** [comment]
I remember a while ago the library not showing up during a fetchland activation in a vintage match. Restarted the program and it happened again in the next match.

I don't remember the exact deck, but there might have been a Fire // Ice in there.

**Comment by vaidashwin** [comment]
I've not convinced myself that it's all split cards or only split cards, but that's a case I found where it's reproducible. I tried two different versions of Fire / Ice with the same result. Without having debugged through I can't say for sure but I would guess this issue is a but displaying certain cards in a popup window (either the library window or the scry/ponder/top/etc. window). If Wear / Tear works I guess being split cards is not the commonality.
